flycheck-ocaml.el

```diff
@@ -39,6 +39,8 @@
 (require 'merlin)
 (require 'flycheck)
 
+(defvar merlin-command-priority) ;; enjoy dynamic scoping
```

Comment:
I find this strange.  We already `(require 'merlin)`.  Why do we have to define this variable explicitly?  Does Merlin not define it?